packs/core_scheduled_rules.yml

```diff
@@ -0,0 +1,12 @@
+AnalysisType: pack
+PackID: PantherManaged.ScheduledRules.Core
+Description: This pack groups a core set of scheduled queries and rules
+PackDefinition:
+  IDs:
+    # Queries
+    - CloudTrail.Password.Spraying
+    - VPC.DNS.Tunneling
+    # Rules
+    - CloudTrail.Password.Spraying
+    - VPC.DNS.Tunneling
```

Comment:
Ahh yea... I had no issues adding these rules/queries manually, but I haven't been able to get a hold of a snowflake backed instance with the bulk uploader version that supports scheduled rules/queries - so it might not work.  This also leads to a question I had above -

> naming convention for queries and their associated rules

These are inherently are "detecting" the same thing, so its hard to come up with two unique names that convey the same idea.  We would start pre-pending `Query.` and `Rule.` but that seems redundant...

Reply:
That's a good point. It seems like the rule should get the readable name, since that will be used in the alert output. Maybe scheduled queries can be prepended with `Scheduled.` or `Query.` as you suggest. Or maybe packs will just need an additional field for query IDs, since they're fundamentally a different construct than any of the python stuff.